wikimedia/wikipedia-ios
# Remove zh-hant and zh-cn as variant options, tweak names

**Fixes Phabricator ticket:** 
https://phabricator.wikimedia.org/T276280#6902051
https://phabricator.wikimedia.org/T276280#6947417

### Notes
* Here I am making tweaks to the json files to remove `zh-hant` and `zh-cn` variants, as requested by @carolynlimadeo. I am also adjusting the wording slightly to so the English subtitles on the language list screen match her comment [here](https://phabricator.wikimedia.org/T276280#6902051).

We already have some translations in for the `chinese-variants-alert-body` key that translated all 8 variants, so I am proactively going into the `.strings` files and removing them (thanks to the "\n"s and variant codes it's pretty easy to tease these out). As soon as this is merged I plan to go into translatewiki.net and update the translations there so they don't get rewritten with 8 variants the next time our auto-magical setup creates a PR.

### Test Steps
Do some light regression testing on: 
1. Variant choices presented to the user when fresh installing, confirm it's 6 and not 8.
2. Upgrade from 6.7.4 on a device that has Chinese Traditional in it's device language settings - confirm nothing seems broken and you see the variant modal listing the 6 options.
